spring-boot/src/test/java/org/springframework/boot/context/config/ConfigFileApplicationListenerTests.java

```diff
@@ -606,7 +606,8 @@ public void absoluteResourceDefaultsToFile() throws Exception {
 		String location = new File("src/test/resources/specificlocation.properties")
 				.getAbsolutePath();
 		TestPropertySourceUtils.addInlinedPropertiesToEnvironment(this.environment,
-				"spring.config.location=" + location);
+				// java.util.Properties needs the values escaped
+				"spring.config.location=" + location.replaceAll("\\\\", "\\\\\\\\"));
```

Comment:
```
spring.config.location=C:\Users\Ruben\Documents\........
```

This is not valid, since it will then become

```
"spring.config.location" => "C:UsersRubenDocuments........"
```

So we have to escape the backslash unfortunately.
Could you also include this commit?